underfs/kodo/src/main/java/alluxio/underfs/kodo/KodoClient.java

```diff
@@ -112,6 +114,9 @@ public InputStream getObject(String key, long startPos, long endPos, long conten
                 + String.valueOf(endPos < contentLength ? endPos - 1 : contentLength - 1))
         .addHeader("Host", mDownloadHost).get().build();
     Response response = mOkHttpClient.newCall(request).execute();
+    if (response.code() == 404) {
```

Comment:
nit: instead of of 404 use HttpStatus.NOT_FOUND

Reply:
Changed, thanks